routers/web/repo/setting_actions.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
Could we also move these files into a sub folder like actions?

Reply:
Yes, done in https://github.com/go-gitea/gitea/pull/24200/commits/cf62e7942e476df63f56ce7374ea351ba711fd4d